tools/itsg33-issue-generator/script.py

```diff
@@ -0,0 +1,236 @@
+"""
+The purpose of this script is to take the CSV file with the ITSG-33 controls and create issues in github for each control.
+The script will create the issue title, body, and labels. The body will have the control definition, class, supplemental guidance,
+ references, general guide, suggested placeholder values, profile specific notes, suggested assignment, and support teams.
+"""
+
+import csv
+import requests
+import json
+import os
+import string
+import logging
+from enum import Enum
+
+"""
+env vars:
+REPO = owner/repo
+GITHUB_TOKEN = github token to create issues
+CSV_FILE = path to csv file
+DEBUG = True/False to print debug info
+"""
+REPO = os.getenv("REPO")
+GITHUB_TOKEN = os.getenv("GITHUB_TOKEN")
+CSV_FILE = "controls.csv"
+
+class Header(Enum):
+    FAMILY = 0
+    CONTROL_ID = 1
+    ENHANCEMENT = 2
+    CONTROL_NAME = 3
+    CONTROL_CLASS = 4
+    CONTROL_DEFINITION = 5
+    REFERENCES = 7
+    SUPPLEMENTAL_GUIDANCE = 6
+    IT_SECURITY_FUNCTION = 8
+    IT_OPERATIONS_GROUP = 9
+    IT_PROJECTS = 10
+    PHYSICAL_SECURITY_GROUP = 11
+    PERSONNEL_SECURITY_GROUP = 12
+    LEARNING_CENTER = 13
+    GENERAL_GUIDE = 14
+    SUGGESTED_PRIORITY = 15
+    SUGGESTED_PLACEHOLDER_VALUES = 16
+    PROFILE_SPECIFIC_NOTES = 17
+
+
+"""
+Set logging level
+"""
+logging.basicConfig(level=logging.ERROR, format='%(asctime)s :%(levelname)s:%(funcName)s:%(lineno)d %(message)s', datefmt='%d-%b-%y %H:%M:%S')
```

Comment:
You can then use your fancy new `LOG_LEVEL` global var here:
```suggestion
logging.basicConfig(level=LOG_LEVEL, format='%(asctime)s :%(levelname)s:%(funcName)s:%(lineno)d %(message)s', datefmt='%d-%b-%y %H:%M:%S')
```

Reply:
Done!